torch/quantization/fake_quantize.py

```diff
@@ -243,10 +243,13 @@ def extra_repr(self):
                                             dtype=torch.quint8, qscheme=torch.per_tensor_affine, reduce_range=True)
 default_weight_fake_quant = FakeQuantize.with_args(observer=MovingAverageMinMaxObserver, quant_min=-128, quant_max=127,
                                                    dtype=torch.qint8, qscheme=torch.per_tensor_symmetric, reduce_range=False)
+
+# TODO(future PR): remove these defaults and enforce activation functions
```

Comment:
I think it's fine to have defaults since these are added automatically for certain operators, theses are not configured by user.

Reply:
yeah, it's not technically related to this PR, more of a code style.  We can discuss at a later time, I don't feel too strongly.